pom.xml

```diff
@@ -155,6 +155,35 @@
             <url>http://maven.dans.knaw.nl/releases</url>
         </pluginRepository>
     </pluginRepositories>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
```

Comment:
The groupId can be left out for `maven-*-plugin` plugins.

Reply:
done